structuredefinitions/UKCore-Observation-VitalSigns-RespirationRate.xml

```diff
@@ -17,8 +17,8 @@
       <rank value="1" />
     </telecom>
   </contact>
-  <description value="Defines the additional constraints and extensions on the UK Core Observation Vital Signs profile" />
-  <purpose value="To provide implementers with additional support when implementing respiratory rate monitoring and to provide a consistent structure to how the data is presented." />
+  <description value="Defines the additional constraints and extensions on the UK Core Observation Vital Signs profile, in order to record the clinical observation of a respiratory rate of a patient." />
+  <purpose value="To provide implementers with additional support when implementing respiratory rate monitoring and to provide a consistent structure to how the data is presented.&#xD;&#xA;&#xD;&#xA;The clicnial observations within scope of this profile include all patient respiration, so Unaided, Oxygen Concentrators, Nasal Cannula, CPAP Devices, Endotracheal Tube, and Mechanical Ventilator are within this scope." />
```

Comment:
'so' isnt a professional word imo. Could we change it to 'including' then removed the 'are within this scope' at the end as it's repeating the start of the sentence